### PR TITLE
Update export-user-data-from-project-online.md

### PR DESCRIPTION
**Pull request by NadinMerali**
Adding documentation about an unsupported scenario

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectOnline/export-user-data-from-project-online.md [hunk 1]
```diff
@@ -342,6 +342,7 @@ After you run the ExportProjectUserContent PowerShell script successfully, you w
    
    > [!NOTE]
    > You can open the .mpp file with Project Professional 2019, Project Professional 2016, or the Project Online Desktop client. 
+   > Saving the exported .mpp files back to Project Online or Project server is not supported.
```
Comment (JohanFreelancer9) [inline ProjectOnline/export-user-data-from-project-online.md:345]:
```suggestion
   > Saving the exported .mpp files back to Project Online or Project Server is not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @NadinMerali, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**